docs/mysql.md

```diff
@@ -1,60 +1,11 @@
-# `mysql` Metrics Receiver
+# MySQL
 
-The mysql receiver can retrieve stats from your mysql instance by connecting as a monitoring user.
-
-## Prerequisites
-
-The `mysql` receiver defaults to connecting to a local MySQL server using a Unix socket and Unix authentication as the `root` user.
-
-## Configuration
-
-Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your mysql configuration.
-
-To configure a receiver for your mysql metrics, specify the following fields:
-
-| Field                 | Default                         | Description |
-| ---                   | ---                             | ---         |
-| `type`                | required                        | Must be `mysql`. |
-| `endpoint`            | `/var/run/mysqld/mysqld.sock`   | The hostname:port or socket path used by mysql |
-| `collection_interval` | `60s`                           | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
-| `username`            | `root`                          | The username used to connect to the server. |
-| `password`            |                                 | The password used to connect to the server. |
-
-Example Configuration:
-
-```yaml
-metrics:
-  receivers:
-    mysql_metrics:
-      type: mysql
-  service:
-    pipelines:
-      mysql_pipeline:
-        receivers:
-          - mysql_metrics
-```
-
-TCP connection with a username and password:
-
-```yaml
-metrics:
-  receivers:
-    mysql_metrics:
-      type: mysql 
-      endpoint: localhost:3306
-      collection_interval: 30s
-      password: pwd
-      username: usr
-  service:
-    pipelines:
-      mysql_pipeline:
-        receivers:
-          - mysql_metrics
-```
+Follow https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mysql
+for instructions to collect logs and metrics from this application using Ops Agent.
```

Comment:
Follow the [installation guide](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mysql) for instructions to collect logs and metrics from this application using Ops Agent.

Reply:
Sure thing. Changed.